src/convertToObject.js

```diff
@@ -5,8 +5,48 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = function (sourceString) {
+  const styleObject = {};
+
+  const cleanSourceString = sourceString
+    .replace(/[\t\r]+/g, '')
+    .replace(/\s*;\s*/g, ';')
+    .replace(/\s*:\s*/g, ':')
+    .replace(/;\n/g, ';');
```

Comment:
<img width="808" alt="image" src="https://github.com/user-attachments/assets/84e27498-e5f2-4dfa-adc6-6d29d48aac8b">
you can remove it